src/main/java/seedu/address/model/interview/UniqueInterviewList.java

```diff
@@ -40,6 +40,33 @@ public boolean containsConflict(Interview toCheck) {
         return false;
     }
 
+    /**
+     * Returns true if the interview is passable based off the number of currently
+     * extended offers and position openings.
+     */
+    public boolean isPassableInterview(Interview toPass) {
```

Comment:
Seems redundant to have this method here because there is no reference to the internal list of Interviews. Only the isAcceptableMethod for the Interview in the argument is called. Why not call it in the AcceptInterviewCommand itself?

Reply:
Perhaps it's okay to refactor it away from `UniqueInterviewList` (and maybe even `Addressbook`), but the `Model Manager` should know the operations that is called.